hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/interceptors/AutocreateTenantRequestFilter.java

```diff
@@ -46,14 +51,27 @@
 @ServerInterceptor
 public class AutocreateTenantRequestFilter implements ContainerRequestFilter {
 
+    /* list of URI chunks to which this filter should not be applied */
+    private static final List<String> uriExceptions = Arrays.asList(".*/inventory/status/?", ".*/inventory/ping/?",
+            ".*/inventory/?");
+    private static List<Pattern> uriExceptionPatterns;
+
+    static {
+        try {
+            uriExceptionPatterns = uriExceptions.stream().map(Pattern::compile).collect(Collectors.toList());
+        } catch (Throwable t) {
+            // swallow, we don't want to fail when classloading
```

Comment:
if this fails, and noone knows about it, then won't [this](https://github.com/hawkular/hawkular-inventory/pull/265/files#diff-4b6fb4d559ca774a421a84a5c995b9e5R113) fail later anyway too?